src/Compilers/Core/Portable/RuleSet/RuleSetInclude.cs

```diff
@@ -57,6 +57,12 @@ public RuleSet LoadRuleSet(RuleSet parent)
                 path = GetIncludePath(parent);
                 ruleSet = RuleSetProcessor.LoadFromFile(path);
             }
+            catch (FileNotFoundException)
+            {
+                // The compiler uses the same rule set files as FxCop, but doesn't have all of
```

Comment:
@heejaechang I think we need to take this through ask mode for compat, otherwise it will break the build for existing projects migrated to newer VS.